src/store/features/editors/visualHTML.ts

```diff
@@ -13,14 +17,34 @@ type MoveElement = {
   payload: DropResult
 }
 
+type DeleteElement = {
+  type: typeof EditorActionsTypes.EditorElementDelete
```

Comment:
You don't need the `typeof`, it's already a type :wink:

Reply:
Yes I see, thanks for pointing this out!